deon/assets/checklist.yml

```diff
@@ -67,14 +67,14 @@ sections:
     section_id: E
     lines:
         - line_id: E.1
+          line_summary: Monitoring and evaluation
+          line: How are we planning to monitor and review the model after it is deployed (e.g., performance monitoring, regular audit of random predictions, human review of high-stakes decisions, downstream impacts of wrong or low-confidence decisions, testing for concept drift)?
```

Comment:
```suggestion
          line: How are we planning to monitor and review the model and its impacts after it is deployed (e.g., performance monitoring, regular audit of predictions, human review of high-stakes decisions, downstream impacts of wrong or low-confidence decisions, testing for concept drift)?
```

Reply:
some suggested wording above. also "downstream impacts" needs a verb or noun before it to make sense in context i think (e.g. review of / testing of / measuring / monitoring for)

Reply:
updated 🙇